src/fsharp/TypeChecker.fs

```diff
@@ -14195,14 +14195,15 @@ module EstablishTypeDefinitionCores = begin
             let writeFakeRecordFieldsToSink (fields:RecdField list) =
                 let nenv = envinner.NameEnv
                 // Record fields should be visible from IntelliSense, so add fake names for them (similarly to "let a = ..")
-                for fspec in (fields |> List.filter (fun fspec -> not fspec.IsCompilerGenerated)) do
-                    let info = RecdFieldInfo(thisTyInst, thisTyconRef.MakeNestedRecdFieldRef fspec)
-                    let nenv' = AddFakeNameToNameEnv fspec.Name nenv (Item.RecdField info) 
-                    // Name resolution gives better info for tooltips
-                    let item = FreshenRecdFieldRef cenv.nameResolver m (thisTyconRef.MakeNestedRecdFieldRef fspec)
-                    CallNameResolutionSink cenv.tcSink (fspec.Range,nenv,item,item,ItemOccurence.Binding,envinner.DisplayEnv,ad)
-                    // Environment is needed for completions
-                    CallEnvSink cenv.tcSink (fspec.Range, nenv', ad)
+                for fspec in fields do
+                    if not fspec.IsCompilerGenerated then
+                        let info = RecdFieldInfo(thisTyInst, thisTyconRef.MakeNestedRecdFieldRef fspec)
+                        let nenv' = AddFakeNameToNameEnv fspec.Name nenv (Item.RecdField info) 
+                        // Name resolution gives better info for tooltips
+                        let item = FreshenRecdFieldRef cenv.nameResolver m (thisTyconRef.MakeNestedRecdFieldRef fspec)
+                        CallNameResolutionSink cenv.tcSink (fspec.Range,nenv,item,item,ItemOccurence.Binding,envinner.DisplayEnv,ad)
```

Comment:
@vasily-kirichenko Minimizing the diff just helps us be certain nothing is changing besides what's intended in the PR.  Using a separate PR for whitespace/stylistic cleanup is preferred.

I understand that localized use of 2-space indent to minimize a diff is a bit tasteless though.  I don't like it but sometimes do it if it's going to help reviewers accept the diff.  Better diffs from github and other viewers are very helpful of course and in many ways a better solution